react-native-template-reason/src/App.re

```diff
@@ -4,55 +4,48 @@
  *
  * Converted from https://github.com/facebook/react-native/blob/d752446b23e0a12171ca0c01d20f5be625f40455/template/App.js
  */
-open BsReactNative;
+open ReactNative;
 
 let instructions =
-  switch (Platform.os()) {
-  | IOS(_) => "Press Cmd+R to reload,\n" ++ "Cmd+D or shake for dev menu"
-  | Android =>
+  switch (Platform.os) {
+  | ios => "Press Cmd+R to reload,\n" ++ "Cmd+D or shake for dev menu"
+  | android =>
     "Double tap R on your keyboard to reload,\n"
     ++ "Shake or press menu button for dev menu"
-  | exception (Platform.UnknownPlatform(p)) when p == "web" => "Press Cmd+R to reload."
-  | exception (Platform.UnknownPlatform(_p)) => ""
+  | web => "Press Cmd+R to reload."
```

Comment:
This won't work.

You will need to do something like

```
switch (Platform.os) {
  | os when os === Platform.ios => ...
  | os when os === Platform.android => ...
  | os when os === Platform.web => ...
  | _ => ...
}
```

(or use `if`/`else`)

Less convenient than with the old bindings, but that's because of #239.

Reply:
Indeed that's very verbose. 
See https://reasonml-community.github.io/bs-react-native/en/docs/apis/Platform/

Reply:
Fixed!